apps/prairielearn/src/pages/instructorRequestCourse/instructorRequestCourse.ts

```diff
@@ -13,23 +12,28 @@ import * as opsbot from '../../lib/opsbot';
 import * as github from '../../lib/github';
 import { config } from '../../lib/config';
 import { CourseRequestSchema, IdSchema } from '../../lib/db-types';
+import { RequestCourse } from './instructorRequestCourse.html';
 
 const router = express.Router();
 const sql = loadSqlEquiv(__filename);
 
 router.get(
   '/',
   asyncHandler(async (req, res) => {
-    res.locals.navPage = 'request_course';
-    res.locals.course_requests = await queryRows(
+    const course_requests = await queryRows(
       sql.get_requests,
       {
         user_id: res.locals.authn_user.user_id,
       },
       CourseRequestSchema,
```

Comment:
As implied by the other comment, this is the wrong schema for it, as it is missing the `details` column.

Reply:
Assuming the logic is moving to the TS file, this schema can remain, so marking as resolved.

Reply:
I dunno, maybe it's better to add it to the schema (but then the schema won't exactly match the table it was built from). Or maybe we lose the details like the PR implies.

I'm not sure which is better.